src/main/resources/db/changelog/changelog-update.xml

```diff
@@ -1,192 +1,194 @@
-    <include file="db/changelog/20200902123935-add-product-derived-product.xml"/>
-    <include file="db/changelog/20200910161135-purge-stale-quartz-data.xml"/>
-    <include file="db/changelog/20201103112757-purge-current-sca-certs.xml"/>
-    <include file="db/changelog/20201210043533-drop-cp-job-table.xml"/>
-    <include file="db/changelog/20210118141207-create-consumer-activation-key-table.xml"/>
-    <include file="db/changelog/20210127105630-migrate-product-hierarchy.xml"/>
-    <include file="db/changelog/20210211154615-add-owner-last-content-updated-column.xml"/>
-    <include file="db/changelog/20210317160453-drop_pool_provided_product_fks.xml"/>
-    <include file="db/changelog/20210412145432-drop_content_cache.xml"/>
-    <include file="db/changelog/20210503150950-add-consumer-service_type-column.xml"/>
-    <include file="db/changelog/20210622100950-cert_serials_drop_collected_column.xml"/>
-    <include file="db/changelog/20210817145050-add-consumer-environments.xml"/>
-    <include file="db/changelog/20211110083000-disallow_duplicate_versioned_entities.xml"/>
-    <include file="db/changelog/20211123093750-migrate-environment-column.xml"/>
-    <include file="db/changelog/20220117152105-add_system_lock_table.xml"/>
-    <include file="db/changelog/20220314130000-64bit_entity_versions.xml"/>
-    <include file="db/changelog/20220328090000-accommodate_large_overrides.xml"/>
-    <include file="db/changelog/20220329140000-add_consumer_name_to_cp_deleted_consumers.xml"/>
-    <include file="db/changelog/20220329154201-owner-product-key-change.xml"/>
-    <include file="db/changelog/20220420165513-drop_pool_derived_product_constraint.xml"/>
-    <include file="db/changelog/20220517162755-add_owner_product_orphaned_date.xml"/>
-    <include file="db/changelog/20220621122940-add_pool_product_index.xml"/>
-    <include file="db/changelog/20220722000000-add_delete_cascade_to_consumer_fks.xml"/>
-    <include file="db/changelog/20220901113942-delete_unused_table_and_columns.xml"/>
-    <include file="db/changelog/20221114152236-add_delete_cascade_to_content_modified_products.xml"/>
-    <include file="db/changelog/20221117125815-prime_system_locks.xml"/>
-    <include file="db/changelog/20221003160605-remove_direct_ak_product_reference.xml"/>
-    <include file="db/changelog/20221003160623-remove_direct_env_content_reference.xml"/>
-    <include file="db/changelog/20221212140215-remove_prodcont_fk_delete_cascade.xml"/>
-    <include file="db/changelog/20230207161633-clear_entity_versions-1.xml"/>
-    <include file="db/changelog/20230220150433-drop-act-key-env-tables.xml"/>
-    <include file="db/changelog/20230322103117-reduce-consumer-name.xml"/>
-    <include file="db/changelog/20230811183400-add-indexes-certificate-cleanup.xml"/>
-    <include file="db/changelog/20230725171517-add-anonymous-claimed_column.xml"/>
+    <property name="project.name" value="candlepin" />
```

Comment:
Oh boy... what linter caught this file?

Reply:
I think this was a VS Code formatting change haha I usually turn off the formatter, but it looks like it was on at some point